### PR TITLE
fix: pin tileindex-validate argo-tasks version to 3.1.0 TDE-1013

### PR DESCRIPTION
**Pull request by paulfouquet**
#### Motivation

There might be a bug introduced by v3.1.1. While investigating/fixing, pinning tileindex-validate to previous argo-tasks version is safer.

#### Modification

Pin argo-tasks version to v3.1.0 for tileindex-validate.

#### Checklist

_If not applicable, provide explanation of why._

- [ ] Tests updated
- [ ] Docs updated
- [ ] Issue linked in Title

### CODE REVIEW

**Review by amfage — APPROVED** [review]

**Review by l0b0 — COMMENTED** [review]
Do you have a reference to a bug report, or description of how it's breaking our workflows?

**Comment by paulfouquet** [comment]
> Do you have a reference to a bug report, or description of how it's breaking our workflows?

https://linz.slack.com/archives/C01BWPHA1EU/p1708378056475019

**Comment by l0b0** [comment]
> > Do you have a reference to a bug report, or description of how it's breaking our workflows?
> 
> [linz.slack.com/archives/C01BWPHA1EU/p1708378056475019](https://linz.slack.com/archives/C01BWPHA1EU/p1708378056475019)

The relevant info should probably be pulled into this PR, including a comment in `workflows/raster/standardising.yaml` so that we can revert once it's been fixed.

File: workflows/raster/standardising.yaml [hunk 1]
```diff
@@ -27,7 +27,7 @@ spec:
   arguments:
     parameters:
       - name: version_argo_tasks
-        value: 'v3'
+        value: 'v3.1.0'
```
Comment (l0b0) [inline workflows/raster/standardising.yaml:30]: There should probably be a comment here to explain

1. why this change was made, and
2. under what condition it can be reverted.

Comment (paulfouquet) [reply workflows/raster/standardising.yaml:30]: I thought there would be an issue in production with this change. To fix the production in priority and asap as people are using the system to process data I made this change in order to be merged quick. Obviously those blocked the review and it's been almost 3 hours since I opened this PR so there was no point trying to get it merged...
